client/src/components/zone/createZoneButton.tsx

```diff
@@ -31,13 +31,17 @@ const useStyles = makeStyles({
   }
 });
 
+export interface CreateZoneButtonProps {
+  otherZoneNames: string[];
```

Comment:
It's hard for me to know what "other" means just in the context of these files. Is there a more specific way to name the "others"?

Reply:
`takenZoneNames` or `reservedZoneNames` perhaps?

Reply:
It's a list of all names that this new zone cannot be named, since these names are already in use by other zones

Reply:
Oh, I know, `unavailableZoneNames`

Reply:
Fixed